pubspec.lock

```diff
@@ -85,11 +85,23 @@ packages:
       url: "https://pub.dartlang.org"
```

Comment:
Traditionally in Dart the lock file is not checked into the repository and added to `.gitignore` instead. This file is automatically generated from `pubspec.yaml` (which _should_ be checked in) by `flutter packages get`.

Reply:
Good suggestion. Thanks